setup.cfg

```diff
@@ -6,4 +6,13 @@
 max_line_length = 88
 [flake8]
 max_line_length = 88
-max_complexity = 10
\ No newline at end of file
+max_complexity = 10
+ignore = E501,W292,W391,W503,W504
+
+# E501    line too long
```

Comment:
Nicely documented!

Reply:
thanks. I always forget some the error code, so this makes it a bit easier to find the meaning of the codes.